app/src/main/java/net/gsantner/markor/activity/NotificationDialogFragment.java

```diff
@@ -0,0 +1,41 @@
+package net.gsantner.markor.activity;
+
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.app.TimePickerDialog;
+import android.os.Bundle;
+import android.text.format.DateFormat;
+import android.widget.TimePicker;
+import android.widget.Toast;
+
+import java.util.Calendar;
+
+/**
+ * Created by Humza on 10/7/2017.
+ */
+
```

Comment:
I didnt look to much into code, but be sure to use AppCompat and NotificationCompat